commands/generate.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var url = require('url'),
+    path = require('path');
+
+var Q = require('q');
+
+var lib = require('manifoldjs-lib');
+
+var log = lib.log,
+    manifestTools = lib.manifestTools,
+    projectBuilder = lib.projectBuilder,
+    utils = lib.utils;
+
+var build = require('./package');
+
+function getW3cManifest(siteUrl, manifestLocation, callback) {

```

Comment:
In v6, it might be nice to make the callback optional and return a promise here.